app/models/patient.rb

```diff
@@ -20,45 +20,40 @@ class Patient < ApplicationRecord
     end
   end
 
-  validates :monitoring_reason, inclusion: {
-    in: VALID_PATIENT_ENUMS[:monitoring_reason],
-    message: "is not an acceptable value, acceptable values are: '#{VALID_PATIENT_ENUMS[:monitoring_reason].reject(&:blank?).join("', '")}'"
-  }
-  validates :monitoring_plan, inclusion: {
-    in: VALID_PATIENT_ENUMS[:monitoring_plan],
-    message: "is not an acceptable value, acceptable values are: '#{VALID_PATIENT_ENUMS[:monitoring_plan].reject(&:blank?).join("', '")}'"
-  }
-  validates :exposure_risk_assessment, inclusion: {
-    in: VALID_PATIENT_ENUMS[:exposure_risk_assessment],
-    message: "is not an acceptable value, acceptable values are: '#{VALID_PATIENT_ENUMS[:exposure_risk_assessment].reject(&:blank?).join("', '")}'"
-  }
```

Comment:
Now that we're only applying the validations on Import and API, we should put these back and remove the corresponding fields from the enum field array on lines 23-37.

Reply:
Done in https://github.com/SaraAlert/SaraAlert/pull/582/commits/111784ddc5397e42ab85ecf9025f1d41b90a8ce2.